source/resource/qx/tool/bin/download-assets

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
```

Comment:
Could we have a node script here? Node is running on Windows too, bash not directly.

Reply:
Could you add one later? The assets are committed statically anyways, so this script needs to be run very rarely  to update the scripts.

Reply:
(You can download and replace individual assets manually if the need arises, this script was just for convenience)